SDK: Codes for Requesting to display rewarded ads to js bridge. (#43)
rakutentech/js-miniapp: js-miniapp-sdk/CHANGELOG.md

### v1.2.0 (In progress)

- Added support for requesting show ads from the host application to allow displaying Interstitial and Rewarded ads [See here](README.MD#4-Show-Ads)

> - Added support for requesting load ads from the host application to allow loading Interstitial and Rewarded ads [See here](README.MD#5-Load-Ads)

Reviewer: Maybe these should be combined into a single line: `Added support for requesting to load and show ads ....`

### v1.2.0 (In progress)

> - Added support for requesting show ads from the host application to allow loading and showing Interstitial and Rewarded ads [See here](README.MD#4-Show-Ads)

Reviewer 2: ```suggestion
- Added support for requesting the load and display of Interstitial & Rewarded Ads by the host application [See here](README.MD#4-Show-Ads)
```
